File: library/src/main/java/com/thefinestartist/finestwebview/helpers/UnitConverter.java
package com.thefinestartist.finestwebview.helpers;

import android.content.Context;

public class UnitConverter {
  public static int dpToPx(Context context, int dp) {
    return (int) (dp * context.getResources().getDisplayMetrics().density + 0.5f);
  }
}

File: library/src/main/java/com/thefinestartist/finestwebview/helpers/UnitConverter.kt
package com.thefinestartist.finestwebview.helpers

import android.content.Context

object UnitConverter {
  @JvmStatic
  fun dpToPx(context: Context, dp: Int): Int {
    return (dp * context.resources.displayMetrics.density + 0.5f).toInt()
  }
}